source/index.html

```diff
@@ -2,11 +2,160 @@
+            <div class="promo__wrapper-link wrapper__link">
+              <a class="promo__link" href="#">Каталог pro</a>
+            </div>
+          </li>
+        </ul>
+        <div class="promo__advantages">
+          <h3 class="advantages__title">Как это работает</h3>
+          <ul class="advantages__list">
+            <li class="advantages__item">
+              <p class="advantages__text">
+                Функциональное питание содержит только полезные питательные вещества.
+              </p>
+            </li>
+            <li class="advantages__item">
+              <p class="advantages__text">
+                Выпускается в виде порошка, который нужно лишь залить кипятком и готово.
+              </p>
+            </li>
+            <li class="advantages__item">
+              <p class="advantages__text">
+                Замените один-два приема обычной еды на наше функциональное питание.
+              </p>
+            </li>
+            <li class="advantages__item">
+              <p class="advantages__text">
+                Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!
+              </p>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="section__example">
+        <h2 class="example__title">Живой пример</h2>
+        <p class="example__text">
+          Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день.
+        </p>
+        <dl class="example__list">
+          <dt class="example__list-term">5 кг</dt>
+          <dd class="example__list-definition">снижение веса</dd>
+          <dt class="example__list-term">60 дней</dt>
+          <dd class="example__list-definition">затрачено времени</dd>
+        </dl>
+        <p class="example__total">Затраты на питание: 15 000 РУБ.</p>
+        <div class="example__wrapper-img">
+          <img class="example__img" width="280" height="232" alt="Фото кота до похудения." src="../source/img/cat_example.png">
+          <div class="example__range">
+            <div class="range__scale">
+              <div class="range__bar">
+                <button class="range__toggle toggle__prew" type="button">
+                  <span class="toggle__label">было</span>
+                </button>
+                <button class="range__toggle toggle__next" type="button">
+                  <span class="toggle__label">стало</span>
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="section__address">
+        <h2 class="address__title">Приглашаем к сотрудничеству дилеров!</h2>
+        <a class="address__link" href="#">ул. Большая Конюшенная, д. 19/8 Санкт-Петербург</a>
```

Comment:
Можно ссылку сразу засунуть
```suggestion
        <a class="address__link" href="https://goo.gl/maps/jXkocHRHetL6EzoB9">ул. Большая Конюшенная, д. 19/8 Санкт-Петербург</a>
```